docs/production/install.md

```diff
@@ -57,67 +65,70 @@ sudo -s  # If not already root
+For more information, see [installer details](#details-what-the-installer-does)
+and [troubleshooting](#troubleshooting) below.
 
 #### Installer options
 
-- `--email=you@example.com`: The email address for the person or team who
-  maintains the Zulip installation. Note that this is a public-facing email
-  address; it may appear on 404 pages, is used as the sender's address for many
-  automated emails, and is advertised as a support address. An email address
-  like support@example.com is totally reasonable, as is admin@example.com. Do
-  not put a display name; e.g. "support@example.com", not "Zulip Support
-  <support@example.com>". This becomes `ZULIP_ADMINISTRATOR`
-  ([docs][doc-settings]) in the Zulip settings.
+- `--email=it-team@example.com`: The email address for the **person or team who
+  maintains the Zulip installation**. Zulip users on your server will see this
+  as the contact email in automated emails, on help pages, on error pages, etc.
+  You can later configure a display name for your contact email with the
+  `ZULIP_ADMINISTRATOR` [setting][doc-settings].
 
-- `--hostname=zulip.example.com`: The user-accessible domain name for
-  this Zulip server, i.e., what users will type in their web browser.
-  This becomes `EXTERNAL_HOST` ([docs][doc-settings]) in the Zulip
-  settings.
+- `--hostname=zulip.example.com`: The user-accessible domain name for this Zulip
+  server, i.e., what users will type in their web browser. This becomes
+  `EXTERNAL_HOST` in the Zulip [settings][doc-settings].
+
+- `--certbot`: With this option, the Zulip installer automatically obtains an
+  SSL certificate for the server [using Certbot][doc-certbot], and configures a
+  cron job to renew the certificate automatically. If you prefer to acquire an
+  SSL certificate another way, it's easy to [provide it to
+  Zulip][doc-ssl-manual].
 
 - `--self-signed-cert`: With this option, the Zulip installer
   generates a self-signed SSL certificate for the server. This isn't
   suitable for production use, but may be convenient for testing.
 
-- `--certbot`: With this option, the Zulip installer automatically
-  obtains an SSL certificate for the server [using
-  Certbot][doc-certbot], and configures a cron job to renew the
-  certificate automatically. If you'd prefer to acquire an SSL
-  certificate yourself in any other way, it's easy to [provide it to
-  Zulip][doc-ssl-manual].
-
-You can see the more advanced installer options in our [deployment options][doc-deployment-options]
+For advanced installer options, see our [deployment options][doc-deployment-options]
 documentation.
 
+:::{important}
+
+If you are [restoring a backup][zulip-backups] or importing data, stop here and
+return to the import instructions for [Slack][slack-import],
+[Mattermost][mattermost-import], [Rocket.Chat][rocketchat-import],
+[Gitter][gitter-import], [Zulip Cloud][zulip-cloud-import], or [another Zulip
+server][zulip-server-import].
```

Comment:
This reads a little confusing if you're restoring a backup, since the "stop here and return to" clause is a bit misplaced.